docs/docs/rest-api/public/api/v2/types/pod-instance-status.raml

```diff
@@ -0,0 +1,62 @@
+#%RAML 1.0 DataType
+type: object
+additionalProperties: false
+properties:
+  id:
+    type: !include name.raml
+    description:|
+      Unique ID of this pod instance in the cluster.
+      TODO(jdef) Probably represents the Mesos executor ID.
+  status:
+    type: string
+    enum: [ PENDING, STAGING, STABLE, DEGRADED, TERMINAL ]
+    description:|
+      PENDING  - Instance is queued for launch.
+      STAGING  - Instance has been launched but is not yet running.
+      STABLE   - Instance is running and, if health checks were specified, is healthy.
+      DEGRADED - One or more of the instance's containers are not stable.
```

Comment:
actually, each container can have a health check. so a single pod instance can have multiple health checks. STABLE pod instances have containers with passing (or none at all) health checks. DEGRADED pod instances have at least one container w/ a failing health check.

i'm happy to change the name to UNHEALTHY if this is a naming thing